qa/evil-tests/src/test/java/org/elasticsearch/bootstrap/EvilBootstrapChecksTests.java

```diff
@@ -28,6 +29,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 
+@ESTestCase.NoSecurityManager
```

Comment:
Why don't we import the annotation itself here so this isn't so verbose?

Reply:
Just saves an extra import :)